doc/releases/changelog-dev.md

```diff
@@ -372,12 +394,11 @@
   [0.+0.j 0.+0.j 0.+0.j 1.-1.j]]
   ```
 
-* `qml.Barrier` with `only_visual=True` now simplifies, via `op.simplify()` to the identity
+* `qml.Barrier` with `only_visual=True` now simplifies via `op.simplify()` to the identity
   or a product of identities.
   [(#3016)](https://github.com/PennyLaneAI/pennylane/pull/3016)
 
-* `__repr__` and `label` methods are more correct and meaningful for Operators with an arithmetic
-  depth greater than 0. The `__repr__` for `Controlled` show `control_wires` instead of `wires`.
+* More correct and intuitive outputs for printing some operators have been added.
   [(#3013)](https://github.com/PennyLaneAI/pennylane/pull/3013)
 
 * Use `Operator.hash` instead of `Operator.matrix` to cache the eigendecomposition results in `Prod` and
```

Comment:
Confused by this 🤔

Reply:
Here they are caching the results of the matrix of sum/prod of operators, this is saved not with the matrix but with the operator hash now. Users don't use it directly.